docs/TektonChain.md

```diff
@@ -122,13 +122,13 @@ spec:
   transparency.url: #value
 ```
 - `disabled` : if the value set as `true`, chains feature will be disabled (default: `false`)
-- `generateSigningSecret`: If set to true, the operator will generate an ECDSA key pair (x509.pem private key and x509-pub.pem public key) and store it in the "signing-secrets" secret in the "tekton-pipelines" namespace. the secret is used by chains controller to sign tekton artifacts(taskruns, pipelineruns). it is important to mention that: 
- * The user should retrieve and store in a safe place the x509-pub.pem public key to verify later artifact attestations.
+- `generateSigningSecret`: When set to true, the operator will generate a cosign key pair (`cosign.key` as the private key  and `cosign.pub` as the public key) and store them in the signing-secrets secret within the tekton-pipelines namespace. This secret is used by the Chains controller to sign Tekton artifacts (taskruns, pipelineruns).
+   Before enabling generateSigningSecret, the signing-secret is empty and does not contain any data. If the signing-secret is empty, enabling generateSigningSecret will create a new Cosign key pair. However, if the secret already contains data, enabling generateSigningSecret will not overwrite the existing secret.. However, if the secret already contains data, enabling generateSigningSecret will not overwrite the existing secret. However, if the secret already contains data, enabling generateSigningSecret should not overwrite the existing secret. It is important to note that:
```

Comment:
this repeated 
However, if the secret already contains data, enabling generateSigningSecret will not overwrite the existing secret. However, if the secret already contains data, enabling generateSigningSecret should not overwrite the existing secret.